AUTHORS

```diff
@@ -10,6 +10,7 @@ Borg authors ("The Borg Collective")
 - Marian Beermann <public@enkore.de>
 - Daniel Reichelt <hacking@nachtgeist.net>
 - Lauri Niskanen <ape@ape3000.com>
+- Abdel-Rahman A. (Abogical)
```

Comment:
I guess your commits contain some email address, would you add that here also?
Do you prefer your family name to be abbreviated?